src/dymaptic.GeoBlazor.Core/Components/Widgets/SliderWidget.cs

```diff
@@ -0,0 +1,1104 @@
+using dymaptic.GeoBlazor.Core.Serialization;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Components.Widgets;
+
+/// <summary>
+///     A slider widget that can be used for filtering data, or gathering numeric input from a user. The slider can have multiple thumbs, and provides you with the ability to format labels and control user input.
```

Comment:
I saw the map image page.  Esri docs had a nice slider "parts" diagram, which would lend itself to enabling a user to do some customization.  I dont know if the slider will need its own "demo" page or not.  But, at least it would assist the user understanding the different pieces and what customizations are possible.

Reply:
I don't think we have an easy way to add images to the docs, but I will add what I almost always remember (but forgot here), a link to that page in the Esri docs.